RRemoteConfig/ConfigModel.swift

```diff
@@ -0,0 +1,7 @@
+internal struct ConfigModel: Decodable {
+    let config: [String: String]
+
+    enum CodingKeys: String, CodingKey {
+        case config = "body"
```

Comment:
So this makes the `body` param get set directly to `config`?

Reply:
Yep, it's quite smart.